cmd/start.go

```diff
@@ -149,8 +150,11 @@ func start(ctx context.Context) {
 
 								start := time.Now()
 
+								ctx, stop := context.WithCancel(ctx)
```

Comment:
Im thinking I should move this child context up so that the `getAvailableJobs` and `startJob` are not using the parent context.

Reply:
It appears that this won't make a difference because both of those requests will have been completed by the time we get to the goroutine where we spawn a child context.